### ISSUE TITLE
Camouflaged close btn of 'Link Copied' dialog box.

### ISSUE THREAD

**Issue by digambar-t7** [EddieHubCommunity/LinkFree#950]
### Description

When we are trying to copy link of any profile to the clipboard, a dialog box pops up. The close btn of this dialog box(green box in the ss) is not visible enough. It has been camouflaged with the background. 

### Screenshots

![Screenshot from 2022-01-10 00-04-15](https://user-images.githubusercontent.com/92970968/148695721-dd36b635-7e69-4241-a1fd-3542883b9e11.png)

**Comment by emmalearnscode** [comment]
I don't think the dialog (toast) has a close button, it's programmed to automatically disappear after a few seconds.

**Comment by digambar-t7** [comment]
Yeah it does closes after few seconds but there's still a close-button. Either we must completely  remove it and let the dialog-box close automatically or must improve the button. Because it points out the incompleteness of the design.

**Comment by emmalearnscode** [comment]
I think I just realised what the problem is and it's hopefully being solved in another pull request. When we updated the icons package to react icons the prime icons got removed. However some of the prime components use prime icons as part of their design and are now not showing correctly. The person working on the profile filter noticed this earlier and has added it back in so hopefully when that pull request gets merged then this problem will be solved too.
